src/main/java/uk/gov/hmcts/reform/managecase/api/payload/CallbackRequest.java

```diff
@@ -11,9 +11,9 @@
 
 @Getter
 @AllArgsConstructor
-@ApiModel("Check Notice of Change Approval Request")
+@ApiModel("About To Submit Callback Request")
```

Comment:
I would use "Callback Request". Reusing this class in ACA-96 and I have the About to Start callback. All callbacks have the same request object.